src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -170,13 +181,31 @@ const OwnerPanel = (props: Props) => {
         </div>
       </div> */}
 
-      {/* hide social links until needed */}
-      {/* <div className={styles.socialContainer}>
-        <a href="#" className={classnames(styles.fa, 'fa fa-twitter')} />
-        <a href="#" className={classnames(styles.fa, 'fa fa-instagram')} />
-        <a href="#" className={classnames(styles.fa, 'fa fa-facebook')} />
-        <a href="#" className={classnames(styles.fa, 'fa fa-youtube')} />
-      </div> */}
+      {
+        <div className={styles.socialContainer}>
+          <div className={styles.socialIconContainer}>
+            <FacebookShareButton
+              url={location}
+              quote={facebookQuote}
+              className="share"
+            >
+              <FacebookIcon size={48} round={true} />
+            </FacebookShareButton>
+          </div>
+
+          <div className={styles.socialIconContainer}>
+            <TwitterShareButton url={location} className="share">
+              <TwitterIcon size={48} round={true} />
+            </TwitterShareButton>
+          </div>
+
+          <div className={styles.socialIconContainer}>
+            <EmailShareButton url={location} className="share">
+              <EmailIcon size={48} round={true} />
+            </EmailShareButton>
+          </div>
+        </div>
+      }
 
```

Comment:
it looks like you're the most familiar with the new Owner Panel. So soon we should convert the styles to styled-components. In the next couple of prs you should see how i do styled components for components that apply for just that page. If we find ourselves reusing code then we can do another type of style of creating styled-components. But this is great!

Reply:
Sounds good!